core/station/impl/src/core/validation.rs

```diff
@@ -188,7 +192,7 @@ impl EnsureExternalCanister {
         principal: Principal,
     ) -> Result<(), ExternalCanisterValidationError> {
         if principal == Principal::management_canister()
-            || principal == ic_cdk::id()
+            || principal == ic_cdk::api::id()
             || principal == SYSTEM_SERVICE.get_upgrader_canister_id()
```

Comment:
we should also add here the ICP ledger which is the only ledger we support currently

Reply:
Done: https://github.com/dfinity/orbit/pull/281/commits/1a5126cc7ab81d6cb7a6d1cbdf4ed60208d66444